cmd/wallet_migrate.go

```diff
@@ -0,0 +1,84 @@
+/*
+Copyright © 2023 Glif LTD
+*/
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/glifio/cli/util"
+	"github.com/spf13/cobra"
+)
+
+// migrateCmd represents the migrate command
+var migrateCmd = &cobra.Command{
+	Use:   "migrate",
+	Short: "Migrates keys from keys.toml to the new encrypted keystore",
+	Run: func(cmd *cobra.Command, args []string) {
+		err := checkWalletMigrated()
+		if err == nil {
+			fmt.Println("Wallet already migrated to encrypted keystore.")
+			return
+		}
+
+		err = migrateLegacyKeys()
+		if err != nil {
+			logFatal(err)
+		}
+		fmt.Printf("Keys successfully migrated to the encrypted keystore!\n\n")
+
+		fmt.Printf("Please set a new passphrase to encrypt the owner key:\n\n")
+
+		as := util.AgentStore()
+		ownerAddr, _, err := as.GetAddrs(util.OwnerKey)
```

Comment:
Same note about accessing the owner off the agent store here as a comment below

Reply:
Same feedback. I'm open to another design, but I don't really have a good alternative idea for it.